README.md

```diff
@@ -85,57 +85,26 @@ const app = express();
-| customShopDomains               | `(RegExp \| string)[]`   |    No    |      `undefined`      | Use this if you need to allow values other than `myshopify.com`                                                                                                              |
-| billing                         | `BillingConfig`          |    No    |      `undefined`      | Billing configurations. [See documentation](docs/usage/billing.md) for full description                                                                                      |
-| restResources                   | `ShopifyRestResources`   |    No    |           -           | Mounts the given REST resources onto the object. Must use the same version as `apiVersion`. Learn more about [using REST resources](docs/usage/rest.md#using-rest-resources) |
-| logger                          | `{[key: string]: any}`   |    No    |           -           | Tweaks the behaviour of the package's internal logging to make it easier to debug applications                                                                               |
-| &nbsp;&nbsp;logger.log          | `AsyncFunction`          |    No    | `() => Promise<void>` | Async callback function used for logging, which takes in a `LogSeverity` value and a formatted `message`. Defaults to using `console` calls matching the severity parameter  |
-| &nbsp;&nbsp;logger.level        | `LogSeverity`            |    No    |  `LogSeverity.Info`   | Minimum severity for which to trigger the log function                                                                                                                       |
-| &nbsp;&nbsp;logger.httpRequests | `boolean`                |    No    |        `false`        | Whether to log **ALL** HTTP requests made by the package. Only works if `level` is `Debug`                                                                                   |
-| &nbsp;&nbsp;logger.timestamps   | `boolean`                |    No    |        `false`        | Whether to add the current timestamp to every log call                                                                                                                       |
-
-## Features
-
-- [OAuth](docs/usage/oauth.md)
-  - [Start endpoint](docs/usage/oauth.md#start-endpoint)
-  - [Callback endpoint](docs/usage/oauth.md#callback-endpoint)
-  - [Using sessions](docs/usage/oauth.md#using-sessions)
-  - [Detecting scope changes](docs/usage/oauth.md#detecting-scope-changes)
-- [REST Admin API client](docs/usage/rest.md)
-  - [Using REST resources](docs/usage/rest.md#using-rest-resources)
-- [GraphQL Admin API client](docs/usage/graphql.md)
-- [GraphqL Storefront API client](docs/usage/storefront.md)
-- [Webhooks](docs/usage/webhooks.md)
-  - [Register a Webhook](docs/usage/webhooks.md#register-a-webhook)
-  - [Process a Webhook](docs/usage/webhooks.md#process-a-webhook)
-- [Billing](docs/usage/billing.md)
+## Tutorials
```

Comment:
When I look at other library docs, I think the term Guides is a more familiar and common pattern.

```suggestion
## Guides
```

Reply:
With this change, I'd also suggest we change the folder name from `usage` to `guides` to maintain consistency